### PR TITLE
add Twitter Bootstrap 3 support

### PR DESCRIPTION
**Pull request by nicolasgrancher**
Hi,

I've updated your library to support Twitter Bootstrap 3.

### CODE REVIEW

**Comment by Maks3w** [comment]
I think that bts3 PRs should target a new branch in this repo.

**Comment by theUniC** [comment]
Thanks for the PR!

As @Maks3w noted, this PR would introduce serious BC breaks. So it must be done in another branch, specific for Bootstrap 3.

**Comment by nicolasgrancher** [comment]
Hi,

You're right.
I think @Emagister should create that branch so I can PR on it ?

**Comment by Maks3w** [comment]
@theUniC must create the target branch first.

File: Twitter/Bootstrap/Form.php [hunk 1]
```diff
@@ -42,6 +45,27 @@ public function __construct($options = null)
...
+            if(!$element instanceof Zend_Form_Element_Submit
+                    and !$element instanceof Zend_Form_Element_Button
+                    and !$element instanceof Zend_Form_Element_Image
+                    and !$element instanceof Zend_Form_Element_Checkbox
+                    and !$element instanceof Zend_Form_Element_MultiCheckbox
+                    and !$element instanceof Zend_Form_Element_Radio)
+            {
+                $this->_addClassNames(array(
+                    'form-control',
+                    'col-lg-'.$this->_getFieldColSize()
```
Comment (Maks3w) [inline Twitter/Bootstrap/Form.php:65]: Is there some possibility of to have a customizable array of lg, md, xs sizes?

**Comment by Maks3w** [comment]
@Mardagg666 Try to avoid hardcode columns sizes like `lg`. Also rewrite the code acomplish with PSR-2 style guide.

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
https://github.com/fabpot/PHP-CS-Fixer

**Comment by theUniC** [comment]
I've just created the branch `bootstrap3` in order to address this PR.

**Comment by nicolasgrancher** [comment]
Ok thanks.
I've just created the PR to the bootstrap3 branch.

**Pull request closed without merging**